__init__.py

```diff
@@ -1,157 +1,168 @@
 import bpy
 
-from bpy.props import *
+#from bpy.props import __all__
 
-from . import texture_tools
+#from . import texture_tools
 from . import tools_panel
-from . import rename_bones
 from . import operators
-from . import mesh_operators
+#from . import mesh_operators
+from . import selected_bones
+from . import create_facs
+from . import copy_pose
+from . import qc_eyes
+from . import batch_smd_export
+from . import empty_to_bones
+from . import copy_procedural
+from . import setup_armature
+from . import clear_blank_shape_keys
+
+# from lists import *
 
 bl_info = {
-    "name": "RED's Tools",
-    "author": "RED_EYE",
-    "version": (1, 3),
+    "name": "Source Engine Tools",
+    "author": "RED_EYE, Davi (Debiddo) Gooz",
+    "version": (1, 5),
     "blender": (2, 80, 0),
-    "description": "Tools for preparing armature for Source Engine export",
-    "category": "Tools"
+    "location": "View3D > UI > Tool",
+    "description": "Multi-purpose tools for preparing armature for Source Engine export, derivated from the deprecated RED Tool's",
+    'warning': '',
+    'wiki_url': 'https://github.com/LoveRenamon/REDs-Tools/wiki',
```

Comment:
Update url to original repo before merge

Reply:
eh?
I even forgot I did this pull request haha
let this as draft for now, I'll do a bit more changes in a few weeks  which includes the right wiki url from your repo

Reply:
Okay. Can you also keep the original functionality? You removed a lot of utility tools from it